### DISCUSSION TITLE
error viewing stream from a 500

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#559] [General]
Up to this point I have been running slinger on a location away from my home.  Today I picked up a 500 from my home and set it up with slinger.  Much to my surprise, the stream runs with error.  I have no problems at the other location at all.  I am watching it with slingerplayer.  I have attached log and config here.  Please help.  Thanks.
[log.zip](https://github.com/GerryDazoo/Slinger/files/10999046/log.zip)

**Comment by f520k510** [comment]
Hi, Gerry.  Can you please take a look at my case?
It reports some packet size error which I don't know how to debug.

**Reply by GerryDazoo** [reply]
Your slingbox is not responding when the code tries to start the stream. It's timing out waiting for the first video packet.
I haven't seen this before. You can try a quick power cycle. and verify the video source is connected.
